feature/anime/api/src/commonMain/kotlin/com/yamal/feature/anime/api/model/AnimeDetails.kt

```diff
@@ -0,0 +1,241 @@
+import com.yamal.feature.network.api.model.PictureNetwork
+import com.yamal.feature.network.api.model.RelatedAnimeEdge
+import com.yamal.feature.network.api.model.RelatedMangaEdge
+import com.yamal.feature.network.api.model.StartSeason
+import com.yamal.feature.network.api.model.StatisticsNetwork
+import com.yamal.feature.network.api.model.StatsStatus
+
+data class AnimeDetails(
+    val id: Int,
+    val title: String,
+    val mainPicture: Picture?,
+    val alternativeTitles: AlternativeTitles?,
+    val startDate: String?,
+    val endDate: String?,
+    val synopsis: String?,
+    val mean: Float?,
+    val rank: Int?,
+    val popularity: Int?,
+    val numListUsers: Int,
+    val numScoringUsers: Int,
+    val nsfw: String?,
+    val genre: List<Genre>,
+    val createdAt: String,
+    val updatedAt: String,
+    val mediaType: MediaType,
+    val status: String,
+    val myListStatus: MyListStatus?,
+    val numEpisodes: Int,
+    val startSeason: AnimeSeason?,
+    val broadcast: Broadcast?,
+    val source: String?,
+    val averageEpisodeDuration: Int?,
+    val rating: String?,
+    val studios: List<AnimeStudio>,
+    val pictures: List<Picture>,
+    val background: String?,
+    val relatedAnime: List<RelatedItem<GenericAnime>>,
+    val relatedManga: List<RelatedItem<GenericManga>>,
+    val recommendations: List<AnimeRecommendation>,
+    val statistics: Statistics?,
+)
+
+data class AnimeSeason(val year: String, val season: Season) {
+    companion object
+}
+
+data class NamedIdObject(val id: Int, val name: String)
+
+data class AlternativeTitles(
+    val synonyms: List<String>,
+    val en: String?,
+    val ja: String?,
+)
+
+data class Broadcast(
+    val dayOfTheWeek: String,
+    val startTime: String?,
+)
+
+data class AnimeRecommendation(
+    val node: GenericAnime,
+    val numRecommendations: Int,
+)
+
+data class Statistics(
+    val numListUsers: Int,
+    val statsStatus: StatsStatus,
+)
+
+data class RelatedItem<T>(
+    val node: T,
+    val relation: Relation,
+)
+
+data class Relation(
+    val type: RelationType,
+    val formatted: String,
+)
+
+data class MyListStatus(
+    val status: String? = null,
+    val score: Int? = null,
+    val numEpisodeWatched: Int? = null,
+    val isRewatching: Boolean? = null,
+    val startDate: String? = null,
+    val finishDate: String? = null,
+    val priority: Int? = null,
+    val numTimesRewatched: Int? = null,
+    val rewatchValue: Int? = null,
+    val tags: List<String>? = null,
+    val comments: String? = null,
+    val updatedAt: String? = null,
+)
+
+enum class RelationType(val serialName: String) {
+    Sequel("sequel"),
+    Prequel("prequel"),
+    AlternativeSetting("alternative_setting"),
+    AlternativeVersion("alternative_version"),
+    SideStory("side_story"),
+    ParentStory("parent_story"),
+    Summary("summary"),
+    FullStory("full_story"),
+    ;
+}
+
+fun PictureNetwork.toPicture() = Picture(
+    large = this.large ?: "",
```

Comment:
i would keep large nullable if possible, in order to fallback with elvis operator